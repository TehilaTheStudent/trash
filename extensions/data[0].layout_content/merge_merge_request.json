[
  {
    "componentName": "d-form",
    "id": 1748407772337,
    "category": "form",
    "props": {
      "layout": "vertical",
      "ref": "formRef"
    },
    "children": [
      {
        "componentName": "d-form-item",
        "id": 1748407774396,
        "category": "form",
        "props": {
          "label": "Repository ID",
          "field": "Repo_ID",
          "help-tips": "",
          "rules": [
            {
              "required": true,
              "message": "Repository ID value cannot be left blank",
              "trigger": [
                "blur",
                "change"
              ]
            },
            {
              "regex": "",
              "message": "Repository ID Invalid content.",
              "trigger": [
                "blur",
                "change"
              ]
            }
          ],
          "visible": {
            "type": "fixed",
            "conditions": [],
            "value": true
          },
          "disabled": false
        },
        "children": [
          {
            "componentName": "d-input",
            "id": 1748407774395,
            "props": {
              "inputType": "normal",
              "showPlaceholder": "repository ID",
              "isTrim": true,
              "autocomplete": "off",
              "defaultValue": "",
              "disabled": {
                "type": "fixed",
                "conditions": [],
                "value": false
              }
            },
            "category": "form"
          }
        ]
      },
      {
        "componentName": "d-form-item",
        "id": 1748407775567,
        "category": "form",
        "props": {
          "label": "Merge Request ID",
          "field": "MR_ID",
          "help-tips": "",
          "visible": {
            "type": "fixed",
            "conditions": [],
            "value": true
          },
          "disabled": false,
          "rules": [
            {
              "required": true,
              "message": "Merge Request ID value cannot be left blank",
              "trigger": [
                "blur",
                "change"
              ]
            },
            {
              "regex": "",
              "message": "Merge Request ID Invalid content.",
              "trigger": [
                "blur",
                "change"
              ]
            }
          ]
        },
        "children": [
          {
            "componentName": "d-input",
            "id": 1748407775566,
            "props": {
              "inputType": "normal",
              "showPlaceholder": "",
              "isTrim": true,
              "autocomplete": "off",
              "defaultValue": "",
              "disabled": {
                "type": "fixed",
                "conditions": [],
                "value": false
              }
            },
            "category": "form"
          }
        ]
      }
    ]
  }
]