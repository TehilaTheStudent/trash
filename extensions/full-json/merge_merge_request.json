{
  "plugin_name": "merge_merge_request",
  "display_name": "",
  "data": [
    {
      "unique_id": "071b19364b234cf79d8bde20b7fc596f",
      "name": "Repo_ID",
      "default_value": "",
      "plugin_name": "merge_merge_request",
      "version": "0.0.4",
      "type": "d-input",
      "workspace_id": "*",
      "layout_content": "[{\"componentName\":\"d-form\",\"id\":1748407772337,\"category\":\"form\",\"props\":{\"layout\":\"vertical\",\"ref\":\"formRef\"},\"children\":[{\"componentName\":\"d-form-item\",\"id\":1748407774396,\"category\":\"form\",\"props\":{\"label\":\"Repository ID\",\"field\":\"Repo_ID\",\"help-tips\":\"\",\"rules\":[{\"required\":true,\"message\":\"Repository ID value cannot be left blank\",\"trigger\":[\"blur\",\"change\"]},{\"regex\":\"\",\"message\":\"Repository ID Invalid content.\",\"trigger\":[\"blur\",\"change\"]}],\"visible\":{\"type\":\"fixed\",\"conditions\":[],\"value\":true},\"disabled\":false},\"children\":[{\"componentName\":\"d-input\",\"id\":1748407774395,\"props\":{\"inputType\":\"normal\",\"showPlaceholder\":\"repository ID\",\"isTrim\":true,\"autocomplete\":\"off\",\"defaultValue\":\"\",\"disabled\":{\"type\":\"fixed\",\"conditions\":[],\"value\":false}},\"category\":\"form\"}]},{\"componentName\":\"d-form-item\",\"id\":1748407775567,\"category\":\"form\",\"props\":{\"label\":\"Merge Request ID\",\"field\":\"MR_ID\",\"help-tips\":\"\",\"visible\":{\"type\":\"fixed\",\"conditions\":[],\"value\":true},\"disabled\":false,\"rules\":[{\"required\":true,\"message\":\"Merge Request ID value cannot be left blank\",\"trigger\":[\"blur\",\"change\"]},{\"regex\":\"\",\"message\":\"Merge Request ID Invalid content.\",\"trigger\":[\"blur\",\"change\"]}]},\"children\":[{\"componentName\":\"d-input\",\"id\":1748407775566,\"props\":{\"inputType\":\"normal\",\"showPlaceholder\":\"\",\"isTrim\":true,\"autocomplete\":\"off\",\"defaultValue\":\"\",\"disabled\":{\"type\":\"fixed\",\"conditions\":[],\"value\":false}},\"category\":\"form\"}]}]}]",
      "front_data_prop": null,
      "label": "default",
      "description": null,
      "required": null,
      "extend_prop": {
        "api_options": null,
        "api_type": null,
        "show_placeholder": null,
        "options": null,
        "disabled_conditions": null,
        "visible_conditions": null
      },
      "validation": {
        "required_message": null,
        "regex": null,
        "regex_message": null,
        "max_length": null,
        "min_length": null
      }
    },
    {
      "unique_id": "8711689b063345d6bc10b5f7dab389b1",
      "name": "MR_ID",
      "default_value": "",
      "plugin_name": "merge_merge_request",
      "version": "0.0.4",
      "type": "d-input",
      "workspace_id": "*",
      "layout_content": "",
      "front_data_prop": null,
      "label": "default",
      "description": null,
      "required": null,
      "extend_prop": {
        "api_options": null,
        "api_type": null,
        "show_placeholder": null,
        "options": null,
        "disabled_conditions": null,
        "visible_conditions": null
      },
      "validation": {
        "required_message": null,
        "regex": null,
        "regex_message": null,
        "max_length": null,
        "min_length": null
      }
    }
  ]
}