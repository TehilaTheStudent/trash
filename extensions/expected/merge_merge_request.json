[
  {
    "key": "Repo_ID",
    "value": "${CODEHUB_ID}"
  },
  {
    "key": "MR_ID",
    "value": "${MR_ID}"
  }
]